examples/vite/.env.example

```
@@ -1,3 +1,4 @@
 INITIA_TEST_MNEMONIC=test test test test test test test test test test test junk
 INITIA_NETWORK_TYPE=mainnet
 INITIA_ROUTER_API_URL=
+INITIA_DEPOSIT_API_URL=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The INITIA_DEPOSIT_API_URL key should go before the INITIA_NETWORK_TYPE key

(UnorderedKey)